src/components/IncomingObjects/IncomingObjects.js

```diff
@@ -3,24 +3,30 @@ import { getIncomingNearEarthObjects } from '../../apiCalls';
 import { useEffect, useState } from 'react';
 import PropTypes from 'prop-types';
 import EventList from '../EventList/EventList';
+import { nanoid } from 'nanoid';
 
-const IncomingObjects = ({ handleNetworkErrors }) => {
+const IncomingObjects = ({ handleNetworkErrors, handleEvent }) => {
   const [incomingObjects, setIncomingObjects] = useState([]);
   
   useEffect(() => {
     (async() => {
       try {
         const futureDate = setFutureDate();
-        const data = await await getIncomingNearEarthObjects(futureDate)
-        console.log('incoming', data)
-        setIncomingObjects(data.data)
+        const data = await getIncomingNearEarthObjects(futureDate)
+        setIncomingObjects({fields: data.fields, data: data.data})
       } catch (error) {
-        handleNetworkErrors(error)
+        handleNetworkErrors(error);
       }
     })();
   }, []);
 
-  
+  const objectEventList = incomingObjects.data && incomingObjects.data.map((event, i) => {
```

Comment:
when you used `i` here, I thought it'd be used for ID. If you're not using it, you could do away with it.

Reply:
I can, it was there before I installed nanoid, I'll get rid of it.